### ISSUE TITLE
Share URL persistance

### ISSUE THREAD

**Issue by dbutrovski** [TerriaJS/aremi-natmap#190]
A support query came our way, with an user reporting that existing share URL link is no longer working:
http://nationalmap.gov.au/renewables/#share=vYUobz 
It is a link from an article on the Clean Energy Council website:
https://www.cleanenergycouncil.org.au/news/2016/March/wind-resource-map.html
The share URL at end of the article will launch AREMI app and upon accepting the disclaimer, produce the "Couldn't expand URL" error message.
The wind resource map layer is still present in the application, however I suspect there may have been changes to the UI or how content is organised that mean shortened URL no longer works.
The question is should there be some checks in place to verify shortened URL, or to report a more meaningful error e.g. "This link is no longer supported in current version of AREMI", or let people know there is a time limit to how long they should expect these shortened URLs to work?
Anyhow, CEC should perhaps update the link in their article.

**Comment by AnaBelgun** [comment]
Clean Energy Council informed of wrong link on 21 Nov. 
@meh9 is it accurate that share url functionality doesn't work? I didn't think so

**Comment by meh9** [comment]
@dbutrovski 

This is now fixed and the Share URL is working again.

It appears that Google had added a new feature for the old legacy Google URL Shortener service and defaulted it to on. This prevented requests from being returned. I've disabled the feature so that things are working again.

As the old Google Share URL service in TerriaJS was deprecated around July 2016, 16 months ago, support for this service will be on best-effort basis only.

I will notify the CEC in this case that they should update to a new share URL.
